### DISCUSSION TITLE
Total watt hours for phases

### DISCUSSION THREAD

**Discussion by waspie** [emporia-vue-local/esphome#33] [Q&A]
Im very new to ESPHOME so I'm not quite sure what to do. I notice there's no total watt hours to the phases like there are the individual circuits. what configuration change would need to be done for this?

**Comment by flaviut** [comment]
Very easy to do, just add these lines to the config:

```
  - { power_id: phase_a_power, platform: total_daily_energy, accuracy_decimals: 0, name:  "Phase A Daily Energy" }
  - { power_id: phase_b_power, platform: total_daily_energy, accuracy_decimals: 0, name:  "Phase B Daily Energy" }
```

This difference here is that the phase power reading has been put into the `power_id` field.
